amperity_datagrid/source/bridge_snowflake.rst

```diff
@@ -0,0 +1,346 @@
+
+          Open the account selector and browse to the account for which Amperity Bridge will be configured. Hover over the account name to view additional details, and then copy the account identifier.
+
+          The copied identifier contains both organization and account name in the format `organization-name.account-name`.
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 2.
+          :align: left
+          :class: no-scaled-link
+     - .. include:: ../../shared/credentials_settings.rst
+          :start-after: .. credential-snowflake-region-start
+          :end-before: .. credential-snowflake-region-end
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 3.
+          :align: left
+          :class: no-scaled-link
+     - Before you can configure Amperity Bridge for data sharing with Snowflake you must :ref:`create a share in Snowflake <bridge-snowflake-sync-amperity-configure-snowflake-create-share>`, add a secure share identifier to that share, identify the correct account locator, and then add the :ref:`Amperity account locator ID <bridge-snowflake-sync-amperity-configure-snowflake-account-locator>` to the share.
+
+       Use the `CURRENT_ACCOUNT <https://docs.snowflake.com/en/sql-reference/functions/current_account>`__ |ext_link| argument in |destination-name| to return the locator ID for your Snowflake account.
+
+       .. important:: Secure data sharing should :ref:`only share secure views <bridge-snowflake-sync-amperity-configure-snowflake-secure-views>` with Amperity.
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 4.
+          :align: left
+          :class: no-scaled-link
+     - A secure share identifier in Snowflake is a unique string---A-Z, 0-9, $, and _ (underscores)---that is added when creating a share. This string must configured in Amperity to enable the bridge.
+
+       Copy the secure share identifier from the Snowflake user interface, and then share the identifier with the individual who will configure Amperity for secure data sharing with Snowflake.
+
+
+   * - .. image:: ../../images/steps-arrow-off-black.png
+          :width: 60 px
+          :alt: Requirement 5.
+          :align: left
+          :class: no-scaled-link
+     - 
+
+.. bridge-snowflake-sync-amperity-get-details-end
+
+
+.. _bridge-snowflake-sync-amperity-configure-snowflake:
+
+Configure Snowflake
+--------------------------------------------------
+
+.. bridge-snowflake-sync-amperity-configure-snowflake-start
+
+Before you can configure Amperity Bridge for data sharing with Snowflake you must create a share in Snowflake, add a secure share identifier to that share, and then identify the correct account locator.
```

Comment:
identify _and add_ the correct Amperity account locator as a consumer for the share